docs/source/release/0.61.0-notes.rst

```diff
@@ -296,10 +296,37 @@ updated following the switch to use new-style error handling by default.
 
 (`PR-#9799 <https://github.com/numba/numba/pull/9799>`__)
 
+Make Numba internally compliant under "new_style" error handling.
+-----------------------------------------------------------------
+
+Numba now uses ``new_style`` error handling by default with no alternative
+available. Numba's internal code is now compliant with this error handling
+style, this to continue to allow extension writers the ability to add further
+implementations of "overloads" without the compiler encountering "hard errors".
+
+(`PR-#9837 <https://github.com/numba/numba/pull/9837>`__)
+
+Fix for Python 3.13.1 comprehension bytecode change
+---------------------------------------------------
+
+Python 3.13.1 introduces an extra ``GET_ITER`` bytecode in comprehension 
+processing. This change breaks Numba's comprehension support.
+This patch adds logic to ignore the new ``GET_ITER`` bytecode to maintain 
+compatibility. 
+
+(`PR-#9848 <https://github.com/numba/numba/pull/9848>`__)
```

Comment:
This PR seems to be missing from the changelog? It's here and has a snippet, so it should be in the changelog too. Or did I miss it?